src/Components/Container.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import BookForm from './BookForm';
+import BookList from './BookList';
+import Header from './Header';
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Please don't disable the linter, remove this line and fix linter errors manually. It will help you to write clean code.